x/staking/keeper/msg_server.go

```diff
@@ -56,6 +56,11 @@ func (k msgServer) CreateValidator(goCtx context.Context, msg *types.MsgCreateVa
 		)
 	}
 
+	//Uses GT instead of GTE for consistency with other MinSelfDelegation checks
+	if msg.MinSelfDelegation.GT(msg.Value.Amount) {
```

Comment:
Wouldn't `msg.Value.Amount.LT(msg.MinSelfDelegation)` be easier to read and understand?

Reply:
Agreed - changed in recent commit!